src/main/java/org/opentripplanner/model/impl/OtpTransitServiceBuilder.java

```diff
@@ -192,11 +192,11 @@ public EntityById<BoardingArea> getBoardingAreas() {
   }
 
   public EntityById<AreaStop> getLocations() {
```

Comment:
getAreaStops

Reply:
I fixed this as the first commit in Part 3 - This method is soon to be deleted as well.